### PR TITLE
Lint for range map

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

**Comment by Stebalien** [comment]
Nice!

File: actors/builtin/market/market_actor.go [hunk 1]
```diff
@@ -571,10 +571,8 @@ func (a Actor) CronTick(rt Runtime, params *adt.EmptyValue) *adt.EmptyValue {
 			builtin.RequireNoErr(rt, dbe.RemoveAll(i), exitcode.ErrIllegalState, "failed to delete deals from set")
 		}
 
-		// NB: its okay that we're doing a 'random' golang map iteration here
-		// because HAMTs and AMTs are insertion order independent, the same set of
-		// data inserted will always produce the same structure, no matter the order
-		for epoch, deals := range updatesNeeded {
+		// TODO FIXME: iteration over map
```
Comment (Stebalien) [inline actors/builtin/market/market_actor.go:574]: Fixed in https://github.com/filecoin-project/specs-actors/pull/594.

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+linters-settings:
+  custom:
+    nomaprange:
+      path: .nomaprange.so
+      description: Checks for range over maps
+      original-url: github.com/Kubuxu/go-no-map-range
```
Comment (ZenGround0) [inline .golangci.yml:6]: can you help me find source?  If I go to https://github.com/Kubuxu/go-no-map-range I get "no branches"

Comment (Kubuxu) [reply .golangci.yml:6]: Sorry, I've forgot to push a branch (pushed just a tag for go.mod).

File: .golangci.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+linters:
+  enable:
+    - nomaprange
+
+run:
+  skip-dirs-use-default: false
+
+issues:
+  exclude-rules:
+    - path: _test\.go
```
Comment (anorth) [inline .golangci.yml:17]: We could probably ignore `support` directory outright too.

**Comment by anorth** [comment]
This failed for me locally so I had to revert for now, but looking forward to figuring out what when wrong and bringing it back.

**Pull request merged**